src/core/utils/macros.h

```diff
@@ -160,4 +160,18 @@ struct alignas(CACHELINE_SIZE) cache_aligned {
 #endif
 
 
+//------------------------------------------------------------------------------
+// Platform specific pragmas
+//------------------------------------------------------------------------------
+
+#if DT_OS_WINDOWS
+  // Disable C4996 warning ("This function or variable may be unsafe")
+  // issued by MSVC for a fully valid and portable code
+  #pragma warning(disable : 4996)
+
+  // Disable C4127 warning ("consider using 'if constexpr' statement instead")
+  // as 'if constexpr' is not available in C++11
+  #pragma warning(disable : 4127)
+#endif
+
```

Comment:
For other compilers we select which warnings to emit and which to hide in the build script: https://github.com/h2oai/datatable/blob/master/ci/ext.py#L297-L316. We can do the same with MSVC using `/wdnnnn` compile flag(s), see https://docs.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level.

Using pragmas like this is less optimal, because it means that (a) this include file has side-effects, (b) occasionally we include this file for the purpose of its side-effects only, (c) if during a subsequent refactor the include is removed, it would be very difficult to notice any problem (esp. since we don't watch for compile warnings on CI machines).

Reply:
Nice, didn't notice we have this capability in the `ext.py`, let me use it.